pkg/build/controller/controller.go

```diff
@@ -317,6 +329,7 @@ func (bc *BuildPodDeleteController) HandleBuildPodDeletion(pod *kapi.Pod) error
 	if build.Status.Phase != nextStatus {
 		glog.V(4).Infof("Updating build %s/%s status %s -> %s", build.Namespace, build.Name, build.Status.Phase, nextStatus)
 		build.Status.Phase = nextStatus
+		build.Status.Reason = "BuildPodDeleted"
 		build.Status.Message = "The Pod for this Build was deleted before the Build completed."
```

Comment:
`The pod for this build was deleted before the build completed.`

Do not capitalize resource names.